File: cpp/program/Elektronik.cpp
#include <bits/stdc++.h>
using namespace std;

// Class Elektronik
class Elektronik
{
private:
    string id;
    string nama;
    int harga;

public:
    // Constructor without parameter
    Elektronik()
    {
        id = "";
        nama = "";
        harga = 0;
    }

    // Constructor with parameter
    Elektronik(string id, string nama, int harga)
    {
        this->id = id;
        this->nama = nama;
        this->harga = harga;
    }

    // Getter and setter for id
    void setId(string id){this->id = id;}
    string getId(){return id;}

    // Getter and setter for nama
    void setNama(string nama){this->nama = nama;}
    string getNama(){return nama;}

    // Getter and setter for harga
    void setHarga(int harga){this->harga = harga;}
    int getHarga(){return harga;}

    // Destructor
    ~Elektronik(){}
};

File: python/program/Elektronik.py
## Class Elektronik
class Elektronik:
    ## Constructor
    def __init__(self, id="", nama="", harga=0):
        self.id = id
        self.nama = nama
        self.harga = harga
    
    ## Setter and getter for id
    def set_id(self, id): self.id = id
    def get_id(self): return self.id

    ## Setter and getter for nama
    def set_nama(self, nama): self.nama = nama
    def get_nama(self): return self.nama

    ## Setter and getter for harga
    def set_harga(self, harga): self.harga = harga
    def get_harga(self): return self.harga